src/pixiewps-wrapper.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+aaaa=$(date +%Y)
```

Comment:
could you use variable names that document the intent, or at least add comments describing what happens ?